### ISSUE TITLE
Testing or mocking out reddit client

### ISSUE THREAD

**Issue by theredcameron** [neoliberal/user_pinger_2#24]
It is difficult to test the bot when Reddit PRAW and slack logging components are directly implemented within the code. Testing would be easier and safer if we were to add a layer of abstraction to the bot that enabled us to 'mock' the reddit connection and logging.

**Comment by theredcameron** [comment]
Nevermind. I see that has been implemented.

**Comment by theredcameron** [comment]
I realize that the tests still depend on a direct connection to Reddit. This will still need to be 'mocked' out.

**Comment by jenbanim** [comment]
Yeah the entire testing infrastructure is broken and out of date. I'm focusing on features for now, but I'll eventually clean up the codebase
